apollo-federation/src/query_graph/graph_path.rs

```diff
@@ -976,22 +953,22 @@ pub(crate) trait GraphPathTriggerVariant: Eq + Hash + std::fmt::Debug {
         }
     }
 
-    fn get_field_parent_type<'a>(&'a self) -> Option<FieldDefinitionPosition>
```

Comment:
Refactors:
- This is returning the field definition when it should be returning the parent type.